premerge/post-submit-testing.md

```diff
@@ -0,0 +1,114 @@
+# Post Submit Testing
```

Comment:
Try to define everything you use; try to avoid long, run-on sentences.

Reply:
Also, be sure to explain WHY we need something or something is important; don't assume it's obvious to your reader.